Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -1998,3 +1998,18 @@
   - !type:BuyerDepartmentCondition
     whitelist:
     - Medical
+
+- type: listing
+  id: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)